__tests__/implementation.test.js

```diff
@@ -276,4 +276,89 @@ describe("app", () => {
         });
     });
   });
+  describe("GET /api/articles (sorting queries)", () => {
+    it("Should respond 200 ", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+          expect(response.body.articles).toBeInstanceOf(Array);
+          expect(response.body.articles[0]).toHaveProperty("created_at");
+          response.body.articles.forEach((article) => {
+            expect(article).toHaveProperty("author");
+            expect(article).toHaveProperty("title");
+            expect(article).toHaveProperty("article_id");
+            expect(article).toHaveProperty("topic");
+            expect(article).toHaveProperty("created_at");
+            expect(article).toHaveProperty("votes");
+          });
+        });
+    });
+    it("should respond with status 200 and an array of articles sorted by created_at descending by default", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+          expect(response.body.articles).toBeInstanceOf(Array);
+          expect(response.body.articles[0]).toHaveProperty("created_at");
+
+          for (let i = 1; i < response.body.articles.length; i++) {
```

Comment:
please refactor this to use [jest sorted](https://www.npmjs.com/package/jest-sorted)